.percy.yml

```
@@ -0,0 +1,95 @@
+# Percy Configuration for VetMed Tracker
+version: 2
+
+discovery:
+  # Disable network idle detection since we handle loading states explicitly
+  network-idle-timeout: 2000
+  allowed-hostname: localhost
+  allowed-hostname: 127.0.0.1
```

YAMLlint (1.37.1)

[error] duplication of key "allowed-hostname" in mapping

(key-duplicates)